packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -319,7 +334,27 @@ pub async fn list_column_by_id(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.list_column_by_id_calls.inc();
 
-    sqlx::query_as!(Columns, r#"SELECT id AS "id: i64", type_id, column_position, column_name, column_type AS "column_type: String", nullable, graphql_type FROM graph_registry_columns WHERE type_id = $1"#, col_id).fetch_all(conn).await
+    let rows = sqlx::query(
+        "SELECT id, type_id, column_position, column_name, column_type, nullable, graphql_type FROM graph_registry_columns WHERE type_id = $1"
+        )
+        .bind(col_id)
+        .fetch_all(conn)
+        .await?;
+
+    let columns = rows
+        .into_iter()
+        .map(|row| Columns {
+            id: row.get("id"),
```

Comment:
Again here, I think we should do the boring thing and just extract the values (while making the types explicit) _then_ create the struct.


```rust
let value: String = row.get(0);
let MyThing { value };
```

Reply:
Ok